relations/domain.py

```diff
@@ -1,3 +1,43 @@
 """Core domain concepts and rules."""
 
-from typing import NamedTuple
\ No newline at end of file
+from datetime import datetime
+from enum import Enum, auto
+from typing import NamedTuple, Optional
+
+ArXivID = str
+RelationID = str
+
+
+class EPrint(NamedTuple):
+    """Info of E-print."""
+
+    arxiv_id: ArXivID
+    version: int
+
+
+class RelationType(Enum):
+    """Type of relations."""
+
+    ADD = auto()
+    EDIT = auto()
+    DEACTIVATE = auto()
```

Comment:
Should we call this ``SUPPRESS`` for consistency with L43?

Reply:
6d2a39de7dbea994c3f9f515bb57feca3a109ba7